public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -135,21 +135,21 @@ block get-heroes-details
 
     For *now* we get back on familiar ground by immediately by 
     converting that `Observable` to a `Promise` using the `toPromise` operator.
-  +makeExample('toh-6/ts/app/hero.service.ts', 'to-promise')(format=".")
+  +makeExcerpt('app/hero.service.ts', 'to-promise')
```

Comment:
I'm not crazy about adding the big blue title to all the one-line examples. It gives the source of the code more weight than the actual code.